app/src/main/java/org/fossasia/openevent/app/utils/ValidateUtils.java

```diff
@@ -11,6 +11,8 @@
     private static final Pattern VALID_URL_REGEX =
         Pattern.compile("^(https?|ftp|file)://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|]");
 
+    private static final Pattern VALID_YEAR_REGEX = Pattern.compile("^\\d{4}$");
```

Comment:
Regexes are slow. This is a fairly trivial check. Can be done via static checking

Reply:
I think databinding validation will work as well